hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/repository/PrisonerDifferencesRepository.kt

```diff
@@ -13,7 +13,7 @@ import java.util.UUID
 @Repository
 interface PrisonerDifferencesRepository : JpaRepository<PrisonerDifferences, UUID> {
   fun findByNomsNumber(nomsNumber: String): List<PrisonerDifferences>
-  fun findByDateTimeBetween(from: Instant, to: Instant): List<PrisonerDifferences>
+  fun findByLabelAndDateTimeBetween(label: String, from: Instant, to: Instant): List<PrisonerDifferences>
```

Comment:
bit naff that this is a string instead of an enum

Reply:
yes , will use enum.